app/renderer/ui/containers/Main.tsx

```diff
@@ -1,6 +1,10 @@
 import * as React from "react"
 import { RouteComponentProps } from "react-router"
 
+export const PATHS = {
```

Comment:
URLs are now to be defined in [constants](https://github.com/witnet/sheikah/blob/master/app/renderer/constants/urls.ts) URL file

Reply:
:question: How about this one?